Clarify how approved_clients interacts with Accounts Push (#64)
fedidcg/LightweightFedCM: README.md

Sending `Login-Status: logged-out` clears the profile information along with the login status bit.

### `approved_clients`: Partially Supported

By default, FedCM makes a distinction between `sign-up` and `sign-in` via a property in the user profile information, called `approved_clients`, that is received in the [fetch the accounts](https://w3c-fedid.github.io/FedCM/#fetch-the-accounts) step: if the `clientId` passed in the `navigator.credentials.get()` call is not a member of `approved_clients`, it means that this client was never previously approved by the user.

> This functionality should not be impacted by either the "Optional Endpoints" or "Config Push" features, but does have an obvious correct behavior for the "Accounts Push" feature.

Reviewer: does *not*?

> The `approved_clients` property is only useful if it is reasonably fresh. In order for `approved_clients` to be useful with Accounts Push, it would require either:

Reviewer 2: ```suggestion
The `approved_clients` property is only useful if it is reasonably fresh. For `approved_clients` to be useful with Accounts Push, it would also either of the following:
```

Author: Not sure this reads any better. I think you made a typo in there but I'm not sure what you meant to suggest.

Author: Took this suggestion with modifications.

* After a successful sign-up for an RP with a given IdP, the IdP would need to immediately update the stored accounts information.

> * The browser would need to automatically update the stored `approved_clients` list with the supplied `clientId`.

Reviewer 2: ```suggestion
* After a successful sign-up for an RP with a given IdP, the IdP immediately updates the stored accounts information.
* The browser automatically updates the stored `approved_clients` list with the supplied `clientId`.
```

> Given that it is only a slight inconvenience for the user to be presented with the FedCM sign-up UI instead of the sign-in UI, relying only on the browser's connected accounts set should be sufficient and doesn't require any additional implementation on the part of either IdPs or browser implementers.

Reviewer 2: ```suggestion
Given that it is only a slight inconvenience for the user to be presented with the FedCM sign-up UI instead of the sign-in UI, relying only on the browser's connected accounts set ought to be sufficient, i.e., no additional implementation on the part of either IdPs or browser implementers is expected to be needed.
```
